### PR TITLE
[OSDOCS-6213]: Azure CAPI TP supported platform statements

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.14+

Issue:
[OSDOCS-5774](https://issues.redhat.com//browse/OSDOCS-5774) > [OSDOCS-6213](https://issues.redhat.com//browse/OSDOCS-6213)

Link to docs preview:
(This PR contains updates to supported platform statements only)
- [Cluster CAPI Operator](https://60522--docspreview.netlify.app/openshift-enterprise/latest/operators/operator-reference.html#cluster-capi-operator_cluster-operators-ref)
- [Managing machines with the Cluster API](https://60522--docspreview.netlify.app/openshift-enterprise/latest/machine_management/capi-machine-management.html)

QE review:
- [x] QE has approved this change.

Additional information:
Includes a few reworded items from Vale suggestions.

### CODE REVIEW

**Comment by jeana-redhat** [comment]
@RadekManak Hey I am not sure why I didn't tag you for review of this two weeks ago, but this small bit of work for the Azure API Tech Preview is ready for a look :)

**Comment by jeana-redhat** [comment]
@sunzhaohua2 PTAL for QE

File: modules/cluster-capi-operator.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 [NOTE]
 ====
-This Operator is available as a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] for Amazon Web Services (AWS) and Google Cloud Platform (GCP).
+This Operator is available as a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] for Amazon Web Services (AWS), Google Cloud Platform (GCP) and Microsoft Azure clusters.
```
Comment (JoeAldinger) [inline modules/cluster-capi-operator.adoc:10]:
```suggestion
This Operator is available as a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] for Amazon Web Services (AWS), Google Cloud Platform (GCP), and Microsoft Azure clusters.
```

**Review by JoeAldinger — COMMENTED** [review]
small nit

**Pull request merged**